node/_process/streams.ts

```diff
@@ -36,8 +36,9 @@ export const stdin = new Readable({
   emitClose: false,
   read(this: Readable, size: number) {
     const p = Buffer.alloc(size || 16 * 1024);
-    const length = Deno.stdin.readSync(p);
-    this.push(length === null ? null : p.slice(0, length));
+    Deno.stdin.read(p).then((length) => {
+      this.push(length === null ? null : p.slice(0, length));
+    });
```

Comment:
I guess there should be catch handler here, otherwise error in `Deno.stdin.read()` will result in unhandled promise rejection